news/_posts/core-weekly/2022-08-22-coreweekly-33-2022.md

```diff
@@ -0,0 +1,135 @@
+
+Last week there was a significant update of the developer documentation that summarizes [notable changes in PrestaShop 8](https://devdocs.prestashop.com/8/modules/core-updates/8.0/#notable-changes). We highly recommend to check them.
+
+## Releases
+
+* [ps_customersignin](https://github.com/PrestaShop/ps_customersignin): [v2.0.5](https://github.com/PrestaShop/ps_customersignin/releases/tag/v2.0.5)
+
+
+## A quick update about PrestaShop's GitHub issues and pull requests:
+
+- [52 new issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+created%3A2022-08-15..2022-08-21) have been created in the project repositories;
+- [23 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2022-08-15..2022-08-21), including [6 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2022-08-15..2022-08-21) on the core;
+- [69 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2022-08-15..2022-08-21) in the project repositories;
+- [36 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2022-08-15..2022-08-21), including [28 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2022-08-15..2022-08-21).
+        
+
+
+## Code changes in the 'develop' branch
+
+
+### Core
+* [#29367](https://github.com/PrestaShop/PrestaShop/pull/29367): Update the theme argument description. Thank you [@PrestaEdit](https://github.com/PrestaEdit)
+* [#29325](https://github.com/PrestaShop/PrestaShop/pull/29325): Fix exception message while removing configuration. Thank you [@PrestaEdit](https://github.com/PrestaEdit)
+
+
+### Back office
+* [#28872](https://github.com/PrestaShop/PrestaShop/pull/28872): [BO] : Class name must be allowed from controller in select render list column. Thank you [@Amit-Kumar-Tiwari-Webkul](https://github.com/Amit-Kumar-Tiwari-Webkul)
+
+
+## Code changes in the '8.0.x' branch
+
+
+### Back office
+* [#29288](https://github.com/PrestaShop/PrestaShop/pull/29288): Restructurize horizontal migration Bridge directories. Thank you [@zuk3975](https://github.com/zuk3975)
+
+
+### Tests
+* [#28906](https://github.com/PrestaShop/PrestaShop/pull/28906): Functional tests - Fix CRUD address test on FO. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation sources
```

Comment:
@kpodemski If you re-implement the core weekly generator in PHP 😄 a nice-to-have would be to split the docs change in sections for different branches. Today PRs against 1.7.x and 8.x are mixed.